### ISSUE TITLE
DIT with no sigma_D sampling in a evenly-spaced log grid (MODIT)

### ISSUE THREAD

**Issue by HajimeKawahara** [HajimeKawahara/exojax#97]
DIT for a evenly-spaced logarithm wavenumber grid (ESLOG) is straightforward for the framework of exojax, considering the instrument response, RV, and a rotation broadening. 

Based on [the snippet](https://github.com/dcmvdbekerom/discrete-integral-transform/blob/master/demo/discrete_integral_transform_log.py) and with the great help by @dcmvdbekerom, I made [a modified version of DIT](https://github.com/HajimeKawahara/addit/blob/main/src/addit/ditlogst.py) in addit prior to the implementation in exojax. This modified version (herafter tagged by MODIT) assumes

1. Input wavenumber grid evenly spaced in logarithm
2. Common Doppler width normalized by line center (i.e. common T for a single isotope) 

The second assumption can remove the dimension of Doppler width from a DIT grid, which reduces the computational complexity. 

I will implement these features into exojax prior to checking auto-diff and/or HMC works.

**Comment by HajimeKawahara** [comment]
The branch is features/logdit.

**Comment by HajimeKawahara** [comment]
MODIT looks better than DIT+interpolation to the log grid. It is likely the interpolation introduces errors, or a common T in MODIT improves the error (not sure though).

![dit2](https://user-images.githubusercontent.com/15956904/125791189-eca3d484-90b5-4292-aba0-b0e57dac233e.png)
![dit3](https://user-images.githubusercontent.com/15956904/125791203-c98e1635-33d9-4c59-b4d9-c5c493bb5d84.png)

**Comment by HajimeKawahara** [comment]
Emission spectrum and the response matrix worked for MODIT.

![dit_emis](https://user-images.githubusercontent.com/15956904/125879364-4789e394-029d-4baa-9640-160d8bea5587.png)

**Comment by HajimeKawahara** [comment]
The raw spectrum and the response-applied one for CH4 using MODIT.

![moditCH4](https://user-images.githubusercontent.com/15956904/126028873-0e7e5a68-9ab8-4f9a-8b86-6159ab37eedd.png)

**Comment by HajimeKawahara** [comment]
I should check the optimal zero padding (note).

**Comment by HajimeKawahara** [comment]
closed w/ merge of #100

**Comment by HajimeKawahara** [comment]
#156 

## Comparison for the Luhman16A model

![compml](https://user-images.githubusercontent.com/15956904/135994244-8b207d8e-05f5-446f-9705-4ed5ff5f1604.png)
